artifacts/rhdh-plugins-reference/argocd/argocd-plugin-readme.adoc

```diff
@@ -1,322 +1,114 @@
-----
-argocd:
-  username: ${ARGOCD_USERNAME}
-  password: ${ARGOCD_PASSWORD}
-  appLocatorMethods:
-    - type: 'config'
-      instances:
-        - name: argoInstance1
-          url: https://argoInstance1.com
-          token: ${ARGOCD_AUTH_TOKEN} # Token to use to instance 1
-----
---
-
-=== Permissions in ArgoCD plugin
-
-Setting permissions for ArgoCD user account not only reduces the scope but also reduces the functionality of the backend. If you scope the permissions for read-only, then actions including creating, deleting, and resyncing the application will not be available. 
+* Select an appropriate card to view the deployment details (for example, commit message, author name, and deployment history) in a sidebar.
 
-The error handling in ArgoCD is designed to alert you when proper permissions are not in place.
-
-=== Self-signed certificates in ArgoCD plugin
-
-By default, the ArgoCD server generates a self-signed certificate. Once you install the ArgoCD plugin, the deployment of argocd-server can be patched to be insecure using the following command:
-
-[source,console]
-----
-kubectl patch deployment argocd-server --type "json" -p '[{"op":"add","path":"/spec/template/spec/containers/0/command/-","value":"--insecure"}]'
-----
-
-Also, you can use Helm to install the ArgoCD plugin and be insecure by default:
-
-[source,console]
-----
-helm upgrade --install argocd argo/argo-cd \
-  --version 3.33.5 \
-  --set 'server.extraArgs={--insecure}'
-----
-
-== For users
-
-=== Using ArgoCD plugin in {product}
-
-The ArgoCD plugin displays the current state of an application in Backstage Catalog.
-
-.Prerequisites
-
-* Your Backstage application is installed and running.
-* You have installed the ArgoCD plugin. For installation steps, see <<Installation>>.
-
-.Procedure
-
-. Open your Backstage application and go to the *Catalog* page.
-. In the *OVERVIEW* tab, you can see the ArgoCD overview card at the bottom.
 +
---
+image::rhdh-plugins-reference/sidebar.png[Sidebar]
 
-image::argo-cd-plugin-overview-card.png[ArgoCD overview card]
-
-The ArgoCD overview card displays application information including *NAME*, *INSTANCE*, *SYNC STATUS*, *HEALTH STATUS*, and *LAST SYNCED*.
---
-
-. To view the detailed application information, select an application from the ArgoCD overview card.
-+
---
-A pop-up containing detailed information about the application appears, along with a link to ArgoCD dashboard.
++ 
+* Select the link icon (image:rhdh-plugins-reference/link.svg[width=13px, height=13px, align=center]) to open the deployment details in Argo CD.
 
-image::argo-cd-plugin-overview-card-details.png[ArgoCD overview card]
---
+. Select the *Overview* tab and navigate to the Deployment summary section to review the summary of your application's deployment across namespaces. Additionally, select an appropriate Argo CD app to open the deployment details in Argo CD, or select a commit from the Revision column to review the changes in GitLab or GitHub.
```

Comment:
Click the **Overview** tab

If Deployment summary is a heading on the UI, then bold it. Same for Revision.

Reply:
Thank you!
Not sure if I understand this. Deployment summary is a section name that the system displays on the Overview tab. Do we need to bold section names? ..same for column names, like Revision.

Please suggest.

Reply:
Based on this discussion - https://redhat-internal.slack.com/archives/C04P5S34GP2/p1722440235254279
...going with Select.

If you feel this is not right, we can discuss more :)

Reply:
@gtrivedi88 I was basing the recommendation to bold the UI elements on [this guidance](https://redhat-documentation.github.io/supplementary-style-guide/#user-interface-elements), but use your judgement